README.md

```diff
@@ -1,2 +1,105 @@
-# module-ballerinax-oracledb
-Oracle Database Connector for Ballerina
+Ballerina OracleDB Library
+===================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerinax-oracledb/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-oracledb/actions?query=workflow%3ABuild)
+  [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-oracledb.svg)](https://github.com/ballerina-platform/module-ballerinax-oracledb/commits/master)
+  [![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/oracledb.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Foracledb)
+  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+  [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-oracledb/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-oracledb)
+
+The OracleDB library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
+
+This provides the functionality required to access and manipulate data stored in a OracleDB database.  
+
+For more information on the operations supported by the `oracledb:Client`, which include the below, go to [The OracleDB Package](https://ballerina.io/learn/api-docs/ballerina/oracledb/).
+
+- Pooling connections
+- Querying data
+- Inserting data
+- Updating data
+- Deleting data
+- Updating data in batches
+- Executing stored procedures
+- Closing the client
+
+For example demonstrations of the usage, go to [Ballerina By Examples](https://ballerina.io/learn/by-example/oracledb-init-options.html).
+
```

Comment:
Do we have BBE for this?

Reply:
Removed BBE links and kept API doc links. commid: b04466c7e3261891042328ee057f4959563e158f